mreg_cli/commands/host_submodules/core.py

```diff
@@ -88,7 +88,7 @@ def add(args: argparse.Namespace) -> None:
     :param args: argparse.Namespace (name, ip, contact, comment, force, macaddress)
 
     """
-    network_or_ip = args.ip
+    network_or_ip: str = args.ip
```

Comment:
Explictly declaring this type ensures we don't pass around an `Any` arg anywhere we potentially expect a `NetworkOrIP`.